backend/fleet_management/migrations/0016_auto_20190829_2155.py

```diff
@@ -0,0 +1,53 @@
+from django.db import migrations
+from django.contrib.auth.models import Group
+from faker import Faker
+
+from fleet_management.constants import Groups
+from ..models import User, Passenger, Drive
+
+
+def create_users_from_passengers(apps, schema_editor):
+    passenger_group = Group.objects.get(name=Groups.Passenger.name)
+
+    passengers = Passenger.objects.all()
+
+    for index, passenger in enumerate(passengers):
+        fake = Faker()
+        password = fake.password()
```

Comment:
why are you faking passwords? Wouldn't this effect in loosing user account passwords?

Reply:
We are changing passengers (no password) to full users, assigned to the Passenger group

Because we are removing a Passenger model we can't do this change in one step :(
Thus I am breaking this down to two PRs. I am having some problems with the second one though... I will try to sort it out before Wednesday though